examples/nl-core-StrongKidsScore-01.xml

```diff
@@ -0,0 +1,89 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Observation xmlns="http://hl7.org/fhir">
+   <id value="nl-core-StrongKidsScore-01"/>
+   <meta>
+      <profile value="http://nictiz.nl/fhir/StructureDefinition/nl-core-StrongKidsScore"/>
+   </meta>
+   <status value="final"/>
+   <code>
+      <coding>
+         <system value="http://snomed.info/sct"/>
+         <code value="108341000146107"/>
+         <display value="StrongKids-screeningsinstrument"/>
+      </coding>
+   </code>
+   <subject>
+      <reference value="Patient/nl-core-Patient-01"/>
+      <type value="Patient"/>
+      <display value="Patient, Johanna Petronella Maria (Jo) van Putten-van der Giessen"/>
+   </subject>
+   <effectiveDateTime value="2016-12-12"/>
+   <valueInteger value="2"/>
+   <note>
+      <text value="Geen sprake van ondervoeding"/>
+   </note>
+   <component>
+      <code>
+         <coding>
+            <system value="urn:oid:2.16.840.1.113883.2.4.3.11.60.40.4.0.1"/>
+            <code value="4028003"/>
+            <display value="StrongKidsScore GewichtsverliesScore"/>
```

Comment:
I believe strong should be written in capital letters, but I'm not sure because the SNOMED code on.code is written in capital letters.

Reply:
Wordt niet gewijzigd in zib (in 2020-release)